prov/tcp/src/tcpx_ep.c

```diff
@@ -1011,12 +1015,7 @@ static int tcpx_pep_reject(struct fid_pep *pep, fid_t fid_handle,
 
 	ofi_shutdown(handle->sock, SHUT_RDWR);
 	ret = ofi_close_socket(handle->sock);
-	if (ret)
-		return ret;
-
-free:
-	free(handle);
```

Comment:
The user can call reject without allocating an endpoint.  In this case, we'll leak the handle allocation.